pele_platform/Adaptive/interaction_restrictions.py

```diff
@@ -111,3 +110,13 @@ def _create_conditions(self, values, name):
             self.conditions.append(name + " > " + str(values["min"]))
         if "max" in values:
             self.conditions.append(name + " < " + str(values["max"]))
+
+    def fill_template(self, template):
+        """
+        Joins self.conditions with AND and fills the PARAMETERS_CHANGE template string.
+        Returns
+        -------
+            Parameters change string for interaction restrictions, which will later be injected into pele_params.py.
```

Comment:
I prefer to label the returns as well (it looks clearer). So:
```
Returns
-------
interaction_conditions : str
    Parameters change string for interaction restrictions, which will later be injected into pele_params.py.
```

I am also missing some break lines in the docstrings here :)